__tests__/getApiTopics.test.js

```diff
@@ -0,0 +1,39 @@
+const app = require("../app");
+const request = require("supertest");
+const data = require("../db/data/test-data/index");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  test("should respond with a 200 status", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+  test("should respond with an array of topic", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toHaveProperty("topics", expect.any(Array));
+      });
+  });
+  test("Array should contain object containing correct properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(body).toHaveProperty("topics", expect.any(Array));
+        expect(typeof topics).toBe("object");
```

Comment:
As we are asserting in a forEach here, could we add another expect to check the length of the topics array please?